### PR TITLE
Ensure only Major and Minor version numbers are used for filter.

### PR DESCRIPTION
**Pull request by pmclanahan**
If a "29.0.1" is released, and product details were updated, and bart were
restarted, it would look for that full version in the logs. This would likely
miss a lot of log lines as downloads of 29.0 would likely continue. This
should fix that so that it continues to only look for "firefox-29.0" followed
by anything but "b".

@jgmize r?

### CODE REVIEW

File: smithers/smithers/utils.py [hunk 1]
```diff
@@ -39,12 +39,18 @@ def set_process_name(title):
...
     except IOError:
-        return conf.FIREFOX_VERSION
+        return None
+
 
-    return prod_details.get('LATEST_FIREFOX_VERSION', conf.FIREFOX_VERSION)
+def get_firefox_version():
+    """Return latest version from product details JSON."""
+    version = _get_fx_version_from_json()
+    version = version or conf.FIREFOX_VERSION
```
Comment (jgmize) [inline smithers/smithers/utils.py]: nit: why not just put the `or conf.FIREFOX_VERSION` on the line above after the `_get_fx_version_from_json()`?

File: smithers/smithers/utils.py [hunk 2]
```diff
@@ -39,12 +39,18 @@ def set_process_name(title):
...
+    # we don't care about the point release
+    version_parts = version.split('.')
+    return '.'.join(version_parts[:2])
```
Comment (jgmize) [inline smithers/smithers/utils.py]: nit: why bother with the version_parts intermediate variable? Why not just

``` python
return '.'.join(version.split('.')[:2])
```

**Comment by jgmize** [comment]
lgtm, other than a couple nits that you can do with as you please. r+

**Comment by pmclanahan** [comment]
Good suggestions. Thanks for the review.

**Pull request merged**